custom/audit.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Pass-through file for developer convenience.
+module.exports = require('../lighthouse-core/audits/audit');
```

Comment:
is this so you can `class MyAudit extends require('custom/audit')`? I'd rather do just expose on the index.js module so you could do `class MyAudit extends require('lighthouse').Audit`